packages/expo/src/Notifications/ExponentNotifications.web.ts

```diff
@@ -113,4 +118,92 @@ export default {
   async cancelAllScheduledNotificationsAsync(): Promise<void> {
     this.dismissNotification();
   },
+
+  async getExponentPushTokenAsync(): Promise<string> {
+    if (!Constants.manifest.owner || !Constants.manifest.slug) {
+      throw new CodedError(
+        'ERR_WEB_PUSH_NOTIFICATIONS_MISSING_CONFIG',
+        'You must provide `owner` and `slug` in `app.json` to use push notifications on web. Read more here: https://docs.expo.io/versions/latest/guides/using-vapid/.'
+      );
+    }
+
+    const data = await _subscribeUserToPushAsync();
+    const experienceId = `@${Constants.manifest.owner}/${Constants.manifest.slug}`;
+    const tokenArguments: { [key: string]: string } = {
+      deviceId: Constants.installationId,
+      experienceId: experienceId,
+      // Also uses `experienceId` for `appId` because there's no `appId` for web.
+      appId: experienceId,
+      deviceToken: JSON.stringify(data),
+      type: 'web',
+    };
+
+    // TODO: Use production URL
+    const response = await fetch('http://expo.test/--/api/v2/push/getExpoPushToken', {
+      method: 'POST',
+      body: JSON.stringify(tokenArguments),
+    }).then(response => response.json());
+
+    // TODO: Error handling
+    return response.data.expoPushToken;
+  },
+
+  async getDevicePushTokenAsync(): Promise<{ type: string; data: Object }> {
+    const data = await _subscribeUserToPushAsync();
+    return { type: 'web', data: data };
```

Comment:
```suggestion
    return { type: Platform.OS, data: data };
```
In the future other platforms might fallback to web.